python_predict_client/README.md

```diff
@@ -0,0 +1,29 @@
+# Python Predict Client
+
+It is the independent Python predict client for TensorFlow Serving.
+
+## Installation
+
+Copy the proto files.
+
+```
+cp ../tensorflow_serving/apis/{model,predict,prediction_service}.proto .
+```
+
+Replce the import path.
+
+```
+sed -i "s/tensorflow_serving\/apis\///g" {predict,prediction_service}.proto
```

Comment:
Thanks for the contribution!

Having to `cp` and `sed` source files just to get it to work seems hard to maintain and might be problematic in some cases. This client still uses Tensorflow, so it appears the main benefit of this is being able to call `python ...` without going through Bazel. We already rely on Bazel very heavily in this repo and I think for these example clients, it's not too unreasonable to use bazel-bin/ even for python (as the original clients do), considering that you already need Bazel to build the model_server itself.

I think the best course of action is to comment on the GitHub issue, possibly linking to your external repo that contains an example so others who want to avoid Bazel can use it. In this repo though, we'd like to keep the original client which uses Bazel that's easier for us to test and maintain. We recommend this as the officially supported and tested approach.